SPECS/binutils/binutils.spec

```diff
@@ -30,10 +63,61 @@ Requires:       %{name} = %{version}
 It contains the libraries and header files to create applications
 for handling compiled objects.
 
+%if %{build_cross}
+%package -n cross-%{name}-common
+Summary: Cross-compilation binutils documentation
+BuildArch: noarch
+
+%description -n cross-%{name}-common
+Documentation for the cross-compilation binutils package.
+%endif
+
+%do_package aarch64-linux-gnu %{build_aarch64}
```

Comment:
Just making sure I understand this; 

This invocation of %do_package creates a binutils-aarch64-linux-gnu package.

Reply:
Yes, but only if `%{build_aarch64}` is `1`.

More context: it's an idea I migrated from Fedora, which in our case is supposed to make it easier to add other architectures in the future.